services/autoscaling/src/simcore_service_autoscaling/modules/instrumentation/_models.py

```diff
@@ -16,6 +16,7 @@
 @dataclass(slots=True, kw_only=True)
 class MetricsBase:
     subsystem: str
+    registry: CollectorRegistry
```

Comment:
Q: I thought you mentioned registry is global? if so, why are you passing it?

Reply:
it is not anymore